### PR TITLE
TF-296 Support upload download file on browser

### PR DESCRIPTION
**Pull request by dab246**
https://user-images.githubusercontent.com/80730648/156513006-05a1ccf5-2eb3-41c9-82ac-0c26112a49b6.mov

### CODE REVIEW

File: lib/features/email/presentation/email_controller.dart [hunk 1]
```diff
@@ -269,6 +290,29 @@ class EmailController extends BaseController {
     }
   }
 
+  void downloadAttachmentForWeb(BuildContext context, Attachment attachment) {
+    _showDownloadingFileDialog(context, attachment);
```
Comment (hoangdat) [inline lib/features/email/presentation/email_controller.dart:294]: IMO, we don't need to show loading dialog in browser

**Comment by hoangdat** [comment]
some tests are failed. Please check it @dab246

File: bitrise.yml [hunk 2]
```diff
@@ -85,7 +85,7 @@ workflows:
         - project_location: "$BITRISE_FLUTTER_PROJECT_LOCATION/core"
     - flutter-test@1:
         inputs:
-        - additional_params: "--verbose"
+        - additional_params: "--platform chrome --verbose"
```
Comment (hoangdat) [inline bitrise.yml:88]: 👍

File: env.file [hunk 3]
```diff
@@ -1 +1 @@
-SERVER_URL=https://dev.open-paas.org
\ No newline at end of file
+SERVER_URL=http://localhost
```
Comment (hoangdat) [inline env.file:1]: you should add this file to your global gitignore in local :)

File: test/features/caching/email_cache_client_test.dart [hunk 4]
```diff
@@ -1,3 +1,5 @@
+@TestOn('vm')
```
Comment (hoangdat) [inline test/features/caching/email_cache_client_test.dart:1]: what is it?

Comment (dab246) [reply test/features/caching/email_cache_client_test.dart:1]: Mark this file for testing only on Dart VM. More details here. [https://github.com/dart-lang/test/tree/master/pkgs/test#platform-selectors](https://github.com/dart-lang/test/tree/master/pkgs/test#platform-selectors)

File: bitrise.yml (hunk 2 above)
Comment (hoangdat) [reply bitrise.yml:88]: why need to add this flag?

Comment (dab246) [reply bitrise.yml:88]: we use library `dart.html` to download file on browser. But it not found in dart test. So we need add flag to target to web

**Comment by hoangdat** [comment]
`dart:html` only use in web environment, we should use `universal:html` instead

**Pull request merged**